### DISCUSSION TITLE
API endpoint for starred topics

### DISCUSSION THREAD

**Discussion by shaedrich** [community/community#69442] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

There is an endpoint to retrieve starred repositories: https://api.github.com/users/octocat/starred \
However, I couldn't find a respective endpoint for starred topics.

Am I missing something? If not, I'd love to have such an API endpoint

**Comment by ghostinhershell** [comment]
Hi @shaedrich, 

Thanks so much for reaching out in our Community. Unfortunately there is no way to fetch this via the API currently. However, I will go ahead and file this as a feature request for future reference.

**Reply by shaedrich** [reply]
Thanks a lot, @ghostinhershell!
